src/eventMapping.ts

```diff
@@ -149,16 +151,6 @@ export function getCloudEvent(req: express.Request): CloudEventsContext | null {
     ceSource = `//${resource.service}/${resource.name}`;
   }
 
-  cloudevent = {
-    contenttype: ceContentType,
-    id: context.eventId,
-    specversion: ceSpecversion,
-    time: context.timestamp,
-    data: data,
-    source: ceSource,
-    type: ceType,
-  };
```

Comment:
I believe this introduces a bug.

- `cloudevent.source`
- `cloudevent.type`

...are never set now.

If this bug is not caught by the test, we should add tests to ensure this is caught..